gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -251,23 +227,31 @@ int main(int argc, char* argv[]) {
       return 3;
     }
 
-    auto proto = bhive_importer.BasicBlockProtoFromMachineCode(*bytes);
+    llvm::Expected<std::vector<gematria::DisassembledInstruction>>
+        instructions = gematria::DisassembleAllInstructions(
+            llvm_support->mc_disassembler(), llvm_support->mc_instr_info(),
+            llvm_support->mc_register_info(), llvm_support->mc_subtarget_info(),
+            *inst_printer, 0, *bytes);
 
-    // Check for errors.
-    if (!proto.ok()) {
-      std::cerr << "Failed to disassemble block '" << hex
-                << "': " << proto.status() << "\n";
+    if (!instructions) {
+      std::cerr << "Failed to disassemble block '" << hex << "\n";
```

Comment:
The closing apostrophe is missing (we should have `"'\n"` rather than `"\n"` at the end).